samples/DevTestLabs/Modules/Library/Az.DevTestLabs2.psm1

```diff
@@ -1464,6 +1464,15 @@ function New-AzDtlVm {
     [string]
     $Size,
 
+    [parameter(Mandatory=$false, ValueFromPipelineByPropertyName = $true, HelpMessage="Storage type to use for virtual machine (i.e. Standard, Premium).")]
+    [Validateset('Standard','Premium')]
```

Comment:
Storage type can be "Standard", "StandardSSD", and "Premium"...  Could you update to reflect this?

Reply:
Updated with requested changes